test/number-guess.test.js

```diff
@@ -0,0 +1,36 @@
+import compareNumbers from '../src/number-guess.js';
+
+const test = QUnit.test;
+
+test('checks to see if it is the same', (assert) => {
+    const userGuess = 4;
+    const correctNumber = 4;
+    const expected = 0;
+
+    const result = compareNumbers(correctNumber, userGuess);
+
+    assert.equal(result, expected);
+
+});
+
+test('checks to see if it is too low', (assert) => {
+    const userGuess = 2;
+    const correctNumber = 4;
+    const expected = 1;
+
+    const result = compareNumbers(correctNumber, userGuess);
+
+    assert.equal(result, expected);
+
+});
+
+test('checks to see if it is too low', (assert) => {
```

Comment:
make sure you change the title of your tests to reflect what you're testing! I think you probably just copied down your previous test and forgot to change this, but wanted to point out that this title doesn't reflect what the test is actually doing!